# DP: Modules/Setup.dist: patch to build _hashlib and _ssl extensions statically

--- a/Modules/Setup.dist
+++ b/Modules/Setup.dist
@@ -211,10 +211,7 @@
 
 # Socket module helper for SSL support; you must comment out the other
 # socket line above, and possibly edit the SSL variable:
-#SSL=/usr/local/ssl
-#_ssl _ssl.c \
-#	-DUSE_SSL -I$(SSL)/include -I$(SSL)/include/openssl \
-#	-L$(SSL)/lib -lssl -lcrypto
+#_ssl _ssl.c -lssl -lcrypto
 
 # The crypt module is now disabled by default because it breaks builds
 # on many systems (where -lcrypt is needed), e.g. Linux (I believe).
@@ -257,6 +254,7 @@
 #_sha256 sha256module.c
 #_sha512 sha512module.c
 
+#_hashlib _hashopenssl.c -lssl -lcrypto
 
 # SGI IRIX specific modules -- off by default.
 
